articles/active-directory/external-identities/direct-federation-adfs.md

```diff
@@ -72,7 +72,7 @@ An AD FS server must already be set up and functioning before you begin this pro
 2. In the navigation pane, select **Relying Party Trusts**.
 3. Under **Actions**, select **Add Relying Party Trust**. 
 4. In the **Add Relying Party Trust** wizard, select **Claims aware**, and then select **Start**.
-5. In the **Select Data Source** section, select the check box for **Import data about the relying party published online or on a local network**. Enter this federation metadata URL: `https://nexus.microsoftonline-p.com/federationmetadata/saml20/federationmetadata.xml`. Select **Next**.
+5. In the **Select Data Source** section, select the check box for **Import data about the relying party published online or on a local network**. Enter the federation metadat URl of the Azure Active Directory you are setting up the federation with. This can be found on the Azure portal **Azure Active Directory** > **App registeration** > **Endpoints**> **Federation metadat document**. Select **Next**.
```

Comment:
```suggestion
5. In the **Select Data Source** section, select the option **Import data about the relying party published online or on a local network**. Enter the federation metadata URL of the Azure AD organization you are setting up federation with. The metadata URL can be found in the Azure portal (**Azure Active Directory** > **App registration** > **Endpoints**> **Federation metadata document**). Then select **Next**.
```

Reply:
@namkedia - Please see these suggested edits. Also, do you obtain this Federation metadata document URL in your own Azure AD portal, or do you need to request it from the other organization? We should clarify.